app/controllers/tufts/qr_status_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Tufts
+  class QrStatusController < ApplicationController
+    protect_from_forgery with: :null_session
+
+    def set_status
+      model = ActiveFedora::Base.find(params[:id])
+      model.qr_status = ['Batch Reviewed']
+      model.save
+      render json: { batch_reviewed: true }
+    end
+
+    def status
+      model = ActiveFedora::Base.find(params[:id])
+      status = if model.qr_status == ['Batch Reviewed']
```

Comment:
@no-reply Could you go ahead and merge this? I've created an issue for moving this functionality out of the controller.